diff -ruN disc-cover-1.5.3./disc-cover disc-cover-1.5.3/disc-cover
--- disc-cover-1.5.3./disc-cover	2003-11-09 17:11:34.000000000 +0100
+++ disc-cover-1.5.3/disc-cover	2003-11-09 19:15:56.293998816 +0100
@@ -164,7 +164,7 @@
 
 unless ($inputfile)
 {
-	required Audio::CD;
+	require Audio::CD;
 }
 
 print STDERR "Disc-Cover: running verbose version $version (config $config_version)\n" if ($flag_verbose);
